content/v1.14/release-notes/1.14.0.md

```diff
@@ -0,0 +1,113 @@
+---
+title: v1.14.0
+released: November, 1, 2023
+weight: 10
+---
+
+The v1.14.0 release is a quarterly Crossplane release. 
+
+Read the 
+[Crossplane announcement blog](https://blog.crossplane.io/crossplane-v1-14/). 
+
+### 🎉 Highlights
+<!-- vale write-good.Weasel = NO --> 
+* [Composition functions]({{< ref "../concepts/composition-functions">}}) promoted to `v1beta1`.
+* A new [Provider DeploymentRuntimeConfig]({{<ref "../concepts/providers#controller-configuration">}}) to eventually replace the deprecated `ControllerConfig`. 
+* New [Crossplane CLI]({{<ref "../cli" >}}) for troubleshooting and package management.
+* Support for ordered deletion with [Usages]({{<ref "../concepts/usages">}}).
+* Alpha support for [real time composition]({{<ref "../concepts/pods#enable-real-time-compositions">}})
+
+
+<!--more--> 
+
+### 🚨 Warnings and breaking changes
+
+* Crossplane Helm images have moved from `index.docker.io` to `xpkg.upbound.io`
+* Changes to TLS certificates. Existing users of external secret stores need to
+  manually update their TLS certificates. Read [Crossplane issue #4565](https://github.com/crossplane/crossplane/pull/4656) for more information. 
+* Removed Vault support for External Secret Stores. Crossplane
+  suggests using the [ESS Plugins] as a replacement.
+* Removed the `controllerConfigRef` from the Configuration package
+  and package revision APIs. 
+
+### 💡 New features
+
+What's new in v1.14.0
+
+#### Crossplane CLI
+
+Crossplane v1.14 introduces a rebuilt [Crossplane CLI]({{<ref "../cli" >}}). 
+
+The Crossplane CLI provides features to help troubleshoot Crossplane resources,
+build and publish Crossplane packages and locally test composition functions.
+
+Read the [CLI documentation]({{<ref "../cli" >}}) for information on installing 
+and the full set of
+supported features. 
+
+#### Real time Compositions
+
+This release adds the alpha feature "real time compositions" to Crossplane. 
+
+Real time compositions allow Kubernetes to immediately notify Crossplane of
+changes to composed resources. Crossplane doesn't need to poll resources to
+determine their state. 
+
+Real time compositions dramatically speed up the time Crossplane reacts to
+changes or failures of managed resources.
+
+Real time compositions are an alpha feature and isn't enabled by default.
```

Comment:
```suggestion
Real time compositions is an alpha feature and isn't enabled by default.
```

We've all agreed to use US English even though it is bad and wrong, right? 😉

Reply:
I changed this and cheated

> The real time compositions feature is an alpha feature and isn't enabled by default.